app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -180,7 +180,9 @@ def experience_description(experience)
     end
 
     def references
-      application_form.application_references.selected.map do |reference|
+      application_form.application_references.map do |reference|
+        next unless reference.selected && reference.feedback_status == 'feedback_provided'
```

Comment:
I don't thinks we need `reference.feedback_status == 'feedback_provided'` as that is part of the selected scope: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/models/application_reference.rb#L13

Reply:
we might need to compact references to avoid `nil` values being in the array

Reply:
Thanks @cwrw for review, I found that using the `.selected` scope was what triggered the multiple queries. There's mention of this problem elsewhere in the presenter https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/presenters/vendor_api/single_application_presenter.rb#L225-L227